physionet-django/events/templates/events/email/event_cohost_cohost_status_change.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}{% autoescape off %}{% filter wordwrap:70 %}
```

Comment:
I don't like the names of both of the mail templates:
`event_cohost_cohost_status_change.html` and `event_host_cohost_status_change.html`

They are very confusing and, in my opinion, do not convey the information in the template.

Reply:
This follows a pattern set up by Amit for the rest of the notifications. 
In the context of the pattern, the names make sense. 

That being said, out of context, it does look confusing. I don't have any specific suggestions for the new names.